src/widget/form/genericchatform.cpp

```diff
@@ -25,11 +25,13 @@
 #include <QShortcut>
 #include <QKeyEvent>
 #include <QSplitter>
+#include <QPushButton>
```

Comment:
I tend to sort in alphabetical order and group by framework/namespace, as you did… makes things easier. So it's one line up in this case. Feel free to sort all includes in this file… :)